src/context/AppContext.tsx

```diff
@@ -1,40 +1,30 @@
 import React, { ReactNode, createContext, useContext, useReducer } from "react";
 
-// Define the initial state for your context
+import { useAuthenticator } from "@aws-amplify/ui-react";
```

Comment:
These vendor provided auth hooks are a gift from the heavens.  🙏👍

Reply:
Absolutely.